src/core/geometry/graph/Graph.h

```diff
@@ -20,12 +17,10 @@ class Graph {
     typedef Element<Elem,Bound> GraphElem;
     typedef Element<Bound,Elem> GraphBound;
 
-    Graph();
+    Graph() = default;
```

Comment:
Is it still necessary to have this default ctor?

Reply:
It is. At some point in `Vertices` is being called implicitly the default constructor of `Graph`.